python/lsst/ts/xml/data/sal_interfaces/LEDProjector/LEDProjector_Commands.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl"?>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+  <SALCommand>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_command_switchAllOn</EFDB_Topic>
+    <Description>Switch on all LEDs connected to the labjack.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_command_switchAllOff</EFDB_Topic>
+    <Description>Switch off all LEDs connected to the labjack.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_command_switchOn</EFDB_Topic>
+    <Description>Switch on specific LED, named by serial number or numerical ID.</Description>
+    <item>
+      <EFDB_Name>serialNumber</EFDB_Name>
+      <Description>Serial number of LED to switch on.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>index</EFDB_Name>
+      <Description>Index number of LED to switch on.</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_command_switchOff</EFDB_Topic>
+    <Description>Switch off specific LED, named by serial number or numerical ID.</Description>
+    <item>
+      <EFDB_Name>serialNumber</EFDB_Name>
+      <Description>Serial number of LED to switch off.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>index</EFDB_Name>
+      <Description>Index number of LED to switch off.</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_command_elapsedTimeOn</EFDB_Topic>
```

Comment:
Not sure I understand what this command is supposed to do. Is this a way to query for how long a lamp has been on? Or is it a way to switch on a lamp for a specified amount of time?

Commands are like actions the CSC can take.

If this is supposed to be a way to query the up time, consider something like `LEDProjector_command_showUpTime`, and that should have `index` and `serialNumber` only, I guess. Then, you can have an event that is published as a result of this command with the information you want (e.g. `LEDProjector_logevent_upTime`, with `index`, `serialNumber` and `upTime` as parameters).

Reply:
It was supposed to show uptime, but I'm sure this can live elsewhere, so I'll just remove it.